backend/src/services/UserService.ts

```diff
@@ -1,53 +1,129 @@
-
+/**
+ * -----------------------------------------------------------------------------
+ * Imports
+ * -----------------------------------------------------------------------------
+ */
+import { resolve } from 'path';
 import { Service, Inject } from 'typedi';
 import UserProfileDAO from '../integrations/UserProfileDAO';
 import { IUserProfileInput, IUserProfile } from '../interfaces/IUser';
 import ScaleService from './ScaleService';
 const { v4: uuidv4 } = require('uuid');
 
+/**
+ * -----------------------------------------------------------------------------
+ * Service class
+ * -----------------------------------------------------------------------------
+ */
 @Service()
 class UserService {
+  /**
+   * ---------------------------------------------------------------------------
+   * Attributes
+   * ---------------------------------------------------------------------------
+   */
   userDao: UserProfileDAO;
   @Inject()
```

Comment:
@inject is another way to initialize this variable. userDao is initialized via the constructor. I think we should rewrite so both are either with @inject or via constructor.  Feel free to pick one. If you go for inject the constructor can be removed entirely.

Reply:
what do you mean "another way"? and which variable are you referring to?